src/Config_Command.php

```diff
@@ -127,10 +127,22 @@ public function create( $_, $assoc_args ) {
 			$assoc_args['extra-php'] = file_get_contents( 'php://stdin' );
 		}
 
-		// TODO: adapt more resilient code from wp-admin/setup-config.php
 		if ( ! \WP_CLI\Utils\get_flag_value( $assoc_args, 'skip-salts' ) ) {
-			$assoc_args['keys-and-salts'] = self::_read(
-				'https://api.wordpress.org/secret-key/1.1/salt/' );
+			try {
+				$assoc_args['keys-and-salts'] = true;
+				$assoc_args['auth-key'] = self::unique_key();
+				$assoc_args['secure-auth-key'] = self::unique_key();
+				$assoc_args['logged-in-key'] = self::unique_key();
+				$assoc_args['nonce-key'] = self::unique_key();
+				$assoc_args['auth-salt'] = self::unique_key();
+				$assoc_args['secure-auth-salt'] = self::unique_key();
+				$assoc_args['logged-in-salt'] = self::unique_key();
+				$assoc_args['nonce-salt'] = self::unique_key();
+			} catch ( Exception $e ) {
```

Comment:
I don't follow this `try ... catch` logic. If `random_int()` doesn't exist, which it wouldn't on < PHP 7.0, my assumption is that you'd see a fatal error like we're seeing in the test.

Can you clarify why you're using `try ... catch`? Or, do we need to use `function_exists()` instead?